Resources/Prototypes/SimpleStation14/Entities/Objects/Magic/Medical/asclepius.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  parent: BaseItem
+  id: StaffAsclepius
+  name: staff of Asclepius
+  description: A wooden rod about the size of your forearm with a snake carved around it, winding it's way up the sides of the rod. Something about it seems to inspire in you the responsibilty and duty to help others.
+  components:
+    - type: AsclepiusStaff
+    - type: Sprite
+      sprite: SimpleStation14/Objects/Magic/Medical/asclepius.rsi
+      state: dormant
+    - type: Item
+      heldPrefix: dormant
+      size: 45
+    - type: DynamicPrice
+      price: 10000
```

Comment:
Should probably be a static price as long as it's meant to be a pretty unique item. Might want to lower it though, so that it's not just obviously a better idea to sell it.